.github/workflows/pipelines-root.yml

```diff
@@ -31,7 +31,7 @@ on:
 
 env:
   PIPELINES_CLI_VERSION: v0.21.2
-  PIPELINES_ACTIONS_VERSION: v1.8.0
+  PIPELINES_ACTIONS_VERSION: 2024-07-19_get-job-id
```

Comment:
- [ ] FIXME: Replace with the actions version known after sibling PR is merged and tagged